codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoderTest.java

```diff
@@ -272,26 +273,34 @@ public void dataReadForUnknownStreamShouldApplyFlowControlAndFail() throws Excep
         }
     }
 
+    @Test(expected = Http2Exception.class)
```

Comment:
it would be good to verify which type of exception is being thrown in this case. For example a `StreamException` doesn't seem appropriate in this case, but would still pass the test.

Reply:
+1